jib-core/src/test/java/com/google/cloud/tools/jib/configuration/BuildConfigurationTest.java

```diff
@@ -51,6 +53,8 @@ public void testBuilder() {
     String expectedTargetImageCredentialHelperName = "anotherCredentialHelper";
     RegistryCredentials expectedKnownTargetRegistryCredentials =
         Mockito.mock(RegistryCredentials.class);
+    List<CredentialRetriever> expectedCredentialRetrievers =
```

Comment:
You don't use this variable anywhere than `setCredentialRetrievers`, so the naming seems confusing. Probably just `credentialRetrievers`.